client/ayon_houdini/plugins/publish/save_scene.py

```diff
@@ -16,12 +17,23 @@ def process(self, context):
         # Filename must not have changed since collecting
         host = registered_host()
         current_file = host.get_current_workfile()
-        assert context.data['currentFile'] == current_file, (
-            "Collected filename from current scene name."
-        )
-
+        if context.data['currentFile'] != current_file:
```

Comment:
Same question as mentioned in https://github.com/ynput/ayon-houdini/pull/193/files/a0bb6163672fbd6a6078e8f48e8dc61615150709#r1883402173